sdk/src/wallet/account/operations/syncing/foundries.rs

```diff
@@ -29,7 +29,10 @@ impl Account {
                 task::spawn(async move {
                     match client.foundry_output_id(foundry_id).await {
                         Ok(output_id) => Ok(Some(client.get_output(&output_id).await?)),
-                        Err(crate::client::Error::NotFound(_)) => Ok(None),
+                        Err(crate::client::Error::NoOutput("foundry", id)) => {
+                            log::debug!("{}", crate::client::Error::NoOutput("foundry", id));
```

Comment:
Sorry, I actually meant debug "print" it like this :sweat_smile: because then you also know the type of the id in the error and don't have to add it manually as a second str
```suggestion
                        Err(crate::client::Error::NoOutput(&format!("{id:?")}) => {
```

Reply:
But this is a match statement :) I'm not sure I follow

Reply:
Ah sorry it should be like this when the error is created and not in the match

Reply:
Commented at the correct place